pkg/ottl/ottlfuncs/README.md

```diff
@@ -589,6 +590,61 @@ Examples:
 
 - `ExtractPatterns(body, "^(?P<timestamp>\\w+ \\w+ [0-9]+:[0-9]+:[0-9]+) (?P<hostname>([A-Za-z0-9-_]+)) (?P<process>\\w+)(\\[(?P<pid>\\d+)\\])?: (?P<message>.*)$")`
 
+### ExtractGrokPatterns
+
+`ExtractGrokPatterns(target, pattern, Optional[namedCapturesOnly], Optional[patternDefinitions])`
+
+The `ExtractGrokPatterns` Converter parses unstructured data into a format that is structured and queryable. 
```

Comment:
The go-grok library is mentioned below, but can you either briefly introduce Grok, or link to a page that gives an introduction? I think this would really help users who are unfamiliar with Grok get a sense of whether it makes sense to use this vs. `ExtractPatterns`.

Reply:
i tried to add some clarity without being too chatty, let me know if this level of verbosity works for you

Reply:
That's great, I think it gives a solid overview. Thank you.